library/experiments/hate_speech.py

```diff
@@ -102,23 +102,23 @@
 
 sklearn_autocorrect = create_nlp_sklearn_pipeline(
     title="sklearn_autocorrect",
-    input_data=input_data,
+    input_data=tweet_eval_hate,
     sklearn_config=sklearn_config,
     autocorrect=True,
 )
 
 sklearn_simple_nb = create_nlp_sklearn_pipeline(
     title="sklearn_simple_nb",
-    input_data=input_data,
+    input_data=tweet_eval_hate,
     sklearn_config=sklearn_config_simple_nb,
     autocorrect=False,
 )
 
-random = Pipeline("random", input_data, [RandomModel("random")])
-all_0s = Pipeline("all_0s", input_data, [AllZerosModel("all_0s")])
-all_1s = Pipeline("all_1s", input_data, [AllOnesModel("all_1s")])
+random = Pipeline("random", tweet_eval_hate, [RandomModel("random")])
+all_0s = Pipeline("all_0s", tweet_eval_hate, [AllZerosModel("all_0s")])
```

Comment:
here we only have a single dataloader

Reply:
I haven't fixed all the library examples yet!